src/instructlab/model/evaluate.py

```diff
@@ -545,6 +552,13 @@ def evaluate(
     with contextlib.suppress(ValueError):
         batch_size = int(batch_size)
 
+    # If 'json' format specified, write logs to the file for clean stdout
+    root_logger = logging.getLogger()
```

Comment:
Since root_logger isn't used anywhere else, it should be in this function.

However, we already have a `logger` at the top of this file, it just uses `__name__` as it's input. Since we might need to use a logger to add the logs to the file would we want to use that one or the root_logger?

One thing to note is that when I ran this PR earlier, I noticed that the log output file had the vLLM logs in it.